packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -49,6 +50,7 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
     phoneNumber: '',
     // country data should be fetched before mounting to prevent a second render
     countries: new Countries(this.props.lng),
+    inputPhonePlaceholder: this.props.inputPhonePlaceholder,
```

Comment:
We should rename the prop, how about `initialInputPhonePlaceholder`?

Reply:
Sure, I'm going to change it